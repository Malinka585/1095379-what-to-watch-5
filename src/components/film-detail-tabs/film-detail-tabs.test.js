import React from "react";
import renderer from "react-test-renderer";
import FilmDetailTabs from "./film-detail-tabs";

const noop = () => {};

const film = {
  id: 1,
  name: `The Grand Budapest Hotel`,
  posterImage: `img/the-grand-budapest-hotel-poster.jpg`,
  backgroundImage: `img/the-grand-budapest-hotel-bg.jpg`,
  genre: `Comedy`,
  released: 2014,
};

it(`Should FilmDetailTabs render correctly`, () => {
  const tree = renderer
    .create(
        <FilmDetailTabs
          activeTab={`All genres`}
          onTabClick={noop}
          film={film}
        />

    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
