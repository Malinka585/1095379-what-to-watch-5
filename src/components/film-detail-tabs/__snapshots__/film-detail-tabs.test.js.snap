// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should FilmDetailTabs render correctly 1`] = `
Array [
  <nav
    className="movie-nav movie-card__nav"
  >
    <ul
      className="movie-nav__list"
    >
      <li
        className="movie-nav__item "
      >
        <a
          className="movie-nav__link"
          onClick={[Function]}
        >
          Overview
        </a>
      </li>
      <li
        className="movie-nav__item "
      >
        <a
          className="movie-nav__link"
          onClick={[Function]}
        >
          Details
        </a>
      </li>
      <li
        className="movie-nav__item "
      >
        <a
          className="movie-nav__link"
          onClick={[Function]}
        >
          Reviews
        </a>
      </li>
    </ul>
  </nav>,
  <h1>
    404.
    <br />
    <small>
      Page not found
    </small>
  </h1>,
]
`;
